### DISCUSSION TITLE
Usage with up-line-or-beginning-search and down-line-or-beginning-search

### DISCUSSION THREAD

**Discussion by kareefardi** [marlonrichert/zsh-autocomplete#605] [Q&A]
I used `zstyle ':autocomplete:key-bindings' enabled no` to disable history and do my own binding. As stated here: https://github.com/marlonrichert/zsh-autocomplete/discussions/424#discussioncomment-2444628 autocomplete interferes where each time up/down is pressed the search query end at end of line, if that makes sense.

**Comment by marlonrichert** [comment]
And the solution is in the same thread in this comment: https://github.com/marlonrichert/zsh-autocomplete/discussions/424#discussioncomment-2478594
